tools/build_info.hh

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+#pragma once
+
+#include "build_mode.hh"
+
+namespace tools::build_info {
+enum class build_type { DEBUG = 0, RELEASE, DEV, SANITIZE, COVERAGE, UNKNOWN };
```

Comment:
seems like these names are problematic as they are also macros (maybe lowercase or camelcase would work)?

Reply:
yes, it was a mistake to use upper case.